### ISSUE TITLE
Axis Grid/Frame in 3D

### ISSUE THREAD

**Issue by manerotoni** [napari/napari#4996]
## 🚀 Feature
An Axis Grid/Frame in 3D. This is a standard feature from many visualization software. Here an example from paraview

![image](https://user-images.githubusercontent.com/3718770/187660038-28b1b2a8-c279-4b74-8e14-4566f3ebb179.png)

and Imaris

![image](https://user-images.githubusercontent.com/3718770/187660387-90644f34-b463-4384-b7fe-ec63e26e502c.png)

## Motivation

A frame really helps of get a sense of perspective, orientation and size of the objects.

**Comment by kevinyamauchi** [comment]
Thanks for the feature request, @manerotoni ! @brisvag has a "work in progress" pull request adding bounding boxes. Is this along the lines of what you are looking for? Please feel free to add any comments or suggestions either here or in the pull request!

https://github.com/napari/napari/pull/4894#issuecomment-1203865332

**Comment by brisvag** [comment]
Yes, I would say that axes and grid are a great candidate for `ViewerOverlay`, using that PR's terms :)

**Comment by ambeeblush** [comment]
Hi everyone!! Congratulations on creating Napari, I love this tool and use it daily for my analyses. It would be incredibly useful to have the axes grid visualization tool, any updates?

**Comment by psobolewskiPhD** [comment]
Related, at I2K I got a request for gridlines in 2D mode across the canvas.

**Comment by brisvag** [comment]
I realize I never posted updates here: I did work on this a bit on vispy (https://github.com/vispy/vispy/pull/2630) and tried to tackle a 3D version of the grid visual as well, but it turned out to be harder than I thought. I'll have to pick it up again!
